code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -418,20 +418,19 @@
 	else
 		..()
 
-/mob/living/simple_animal/update_canmove(value_otherwise = TRUE)
-	if(IsUnconscious() || IsStun() || IsKnockdown() || stat || resting)
+/mob/living/simple_animal/update_mobility(value_otherwise = MOBILITY_FLAGS_DEFAULT)
+	if(IsUnconscious() || IsStun() || IsParalyzed() || stat || resting)
 		drop_all_held_items()
-		canmove = FALSE
+		mobility_flags = NONE
 	else if(buckled)
-		canmove = FALSE
+		mobility_flags = ~MOBILITY_MOVE
```

Comment:
should this not be DISABLE_BITFIELD or is there a reason it's specifically everything BUT mobility_move?

Reply:
because simple mobs have a binary stun/nostun state, if they're buckled they can't move but they can do everything else as well, they are not stunned.